dkg-primitives/src/types.rs

```diff
@@ -80,12 +80,15 @@ pub struct DKGSignedPayload<Key> {
 	pub payload: Vec<u8>,
 	/// Signature for the payload
 	pub signature: Vec<u8>,
+	/// Bytes representation of the signature recid
+	pub signature_recid: Vec<u8>,
```

Comment:
It's probably not necessary to have signature_recid, since we already have signature field to contain signature's bytes?

Reply:
The signature field is already converted to the runtime version of ecdsa, but in one of the tests we still needed the actual SignatureRecid, that's why I added them both